webhttpd.c

```diff
@@ -334,6 +334,27 @@ static void response_client(int client_socket, const char *template, char *back)
 }
 
 /**
+ * get_host
+ *      Gets the host (IP) of a client from the socket file descriptor
+ * Returns nothing
+ */
+static void get_host(char *buf, int fd)
```

Comment:
Basically, with one difference: Stream uses `sockaddr_storage` and Webcontrol uses `sockaddr_in6`. I had issues trying to handle both in one function, cleanly.